audio/out/ao_coreaudio_exclusive.c

```diff
@@ -462,10 +462,9 @@ const struct ao_driver audio_out_coreaudio_exclusive = {
         .hog_pid = -1,
         .stream = 0,
         .stream_idx = -1,
-        .changed_mixing = false,
```

Comment:
This technically isn't a option so we probably shouldn't change it (even though it shouldn't matter).